### DISCUSSION TITLE
Where do I learn more about what this tool allows me to do?

### DISCUSSION THREAD

**Discussion by jessegavin** [johnlindquist/kit#34] [Q&A]
I read the following on the website

> Run simple in your shell to launch an interactive prompt to train you to write your own scripts.

I installed simplescripts, and ran simple and was prompted with "What do you want to do?".

I see a menu of options, including a "new" option, but there's no "training" or any other information that explains what I might do with simplescripts.

**Comment by johnlindquist** [comment]
@jessegavin Thanks for dropping in.

I've been waiting to add an official training/docs until I finalize APIs for 1.0. I probably jumped the gun on making the website public, but I've been using it to help people get set up 1 on 1 over Zoom calls to better understand how people would use it and catch bugs.

I'm anticipating a 1.0 next week.

**Comment by johnlindquist** [comment]
In case anyone else stumbles on this, here's 2 streams I've done so far:

https://www.youtube.com/watch?v=TKftp01LYik&ab_channel=JohnLindquist
https://www.youtube.com/watch?v=-NksUwGZ5pc&feature=youtu.be&ab_channel=JohnLindquist
